### PR TITLE
Check neutron router port mtus

### PR DESCRIPTION
**Pull request by dosaboy**
Adds a check to ensure that ml2-ovs router ports have an mtu less than the physical network and raise an OpenstackWarning if any found not to match this. Also adds neutron port mtu info to summary.

Resolves: #747

### CODE REVIEW

File: hotsos/core/plugins/openstack/neutron.py [hunk 1]
```diff
@@ -10,6 +10,11 @@
     OSTServiceBase,
 )
 
+# See https://github.com/openstack/neutron-lib/blob/master/neutron_lib/constants.py#L346  # noqa, pylint: disable=C0301
+IP_HEADER_BYTES = 20
+GRE_HEADER_BYTES = 28
```
Comment (brianphaley) [inline hotsos/core/plugins/openstack/neutron.py:15]: I think this is 22

File: hotsos/plugin_extensions/openstack/service_network_checks.py [hunk 2]
```diff
@@ -69,6 +79,53 @@ def get_phy_port_health_info(self):
...
+
+            for ns in self.cli.ip_netns():
+                # strip index
+                ns = ns.partition(" ")[0]
+                for nsprefix in ['qrouter', 'snat']:
+                    if not ns.startswith(nsprefix):
+                        continue
+
+                    for port in HostNetworkingHelper().get_ns_interfaces(ns):
+                        for pprefix in ['qr']:
```
Comment (brianphaley) [inline hotsos/plugin_extensions/openstack/service_network_checks.py:100]: In the snat- namespaces there will be sg-* interfaces with this reduced mtu as well.

File: hotsos/core/plugins/openstack/neutron.py (hunk 1 above)
Comment (dosaboy) [reply hotsos/core/plugins/openstack/neutron.py:15]: doh! thanks brian

File: hotsos/plugin_extensions/openstack/service_network_checks.py (hunk 2 above)
Comment (dosaboy) [reply hotsos/plugin_extensions/openstack/service_network_checks.py:108]: good call. will fix

**Pull request merged**